cd-scripts/deploy-contracts.sh

```diff
@@ -97,7 +97,9 @@ do
           echo "$chain - $contract : deployment of $contract in $chain"
 
           echo "$chain - $contract : storing contract"
-          CODE_ID=$($BINARY_NAME tx wasm store ../artifacts/$GIT_CONTRACTS_ASSET.wasm --from $KEYRING_KEY_NAME --chain-id $CHAIN_ID   --gas=auto --gas-adjustment 1.2  --gas-prices=$GAS_PRICES$DENOM --broadcast-mode=block --node=$NODE_URL -y |yq -r ".logs[0].events[1].attributes[1].value") #for wasmd 0.29.0-rc2 and maybe above, change attributes[0] --> attributes[1]
+          sleep 10
+          CODE_ID=$($BINARY_NAME tx wasm store ../artifacts/$GIT_CONTRACTS_ASSET.wasm --instantiate-anyof-addresses $DEPLOYMENT_KEY_BECH_ADDR --from $KEYRING_KEY_NAME --chain-id $CHAIN_ID   --gas=auto --gas-adjustment 1.2  --gas-prices=$GAS_PRICES$DENOM --broadcast-mode=block --node=$NODE_URL -y |yq -r ".logs[0].events[1].attributes[1].value") #for wasmd 0.29.0-rc2 and maybe above, change attributes[0] --> attributes[1]
```

Comment:
Can we somehow stop here when we don't get an integer in `CODE_ID`?

Reply:
Oh but what if you get some other event in `yq -r ".logs[0].events[1].attributes[1].value"` such as an address or an amount?

Reply:
okay 🤞

Reply:
Sounds like something we should hardened mid term. There can easily be more event by any kind of Cosmos SDK or wasmd upgrade. But since it worked for testnet, the code ID is likely at the same spot.